### PR TITLE
Tidal typing

### PR DESCRIPTION
**Pull request by jozefKruszynski**
First attempt at fixing a bunch of typing issues

### CODE REVIEW

File: music_assistant/server/providers/tidal/__init__.py [hunk 1]
```diff
@@ -206,7 +209,7 @@ class TidalProvider(MusicProvider):
 
     async def handle_setup(self) -> None:
         """Handle async initialization of the provider."""
-        self._tidal_user_id = self.config.get_value(CONF_USER_ID)
+        self._tidal_user_id = str(self.config.get_value(CONF_USER_ID))
```
Comment (marcelveldt) [inline music_assistant/server/providers/tidal/__init__.py:212]:
```suggestion
        self._tidal_user_id:str = self.config.get_value(CONF_USER_ID)
```

Comment (jozefKruszynski) [reply music_assistant/server/providers/tidal/__init__.py:212]: I'll try to remove all the silly casts

**Comment by marcelveldt** [comment]
@jozefKruszynski can you rebase and recheck this one now that we have the updated linter rules ?

**Comment by jozefKruszynski** [comment]
Will do

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Review by marcelveldt — APPROVED** [review]
nice job!

**Pull request merged**